ana_scans.py

```diff
@@ -0,0 +1,227 @@
+#!/bin/env python
+
+def launchAna(args):
+  return launchAnaArgs(*args)
+
+def launchAnaArgs(anaType, cName, cType, scandate1, scandate2=None, ztrim=4.0, chConfigKnown=False, channels=False, panasonic=False):
+  import os
+  import subprocess
+  from subprocess import CalledProcessError
+  from anaInfo import ana_config
+  from gempython.utils.wrappers import runCommand
+
+  dataPath  = os.getenv('DATA_PATH')
+  dirPath   = ""
+  elogPath  = "%s/%s"%(os.getenv('ELOG_PATH'),scandate1)
+    
+  print "Analysis Requested: %s"%(anaType)
+
+  #Build Commands
+  cmd = ["python"]
```

Comment:
Okay I will try to change this.  

But I think there is still a misconception.  Yes I understand that `vfatqc-python-scripts` needs to be added to $PATH.  My question was _what is doing that_?

The only thing that I know that is the cms-gem-daq-project that sets up the environment is:

`
cmsgemos/setup/paths.sh
`

Are you asking:

`
$BUILD_HOME/vfatqc-python-scripts
`

Be added to the `$PATH` via `cmsgemos/setup/paths.sh`, which would require a separate PR in cmsgemos, or to include some new script that is included in `vfatqc-python-scripts` that the user should launch like paths.sh is launched?